com.mastermindzh.tidal-hifi.yml

```diff
@@ -0,0 +1,58 @@
+---
+app-id: com.mastermindzh.tidal-hifi
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+build-options:
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+  append-path: /usr/lib/sdk/node16/bin
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=pulseaudio'
+  - '--share=network'
+  - '--device=dri'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--filesystem=xdg-run/app/com.discordapp.Discord:create'
+  - '--own-name=org.mpris.MediaPlayer2.tidal-hifi'
+  - '--talk-name=org.freedesktop.Notifications'
+modules:
+  - name: tidal-hifi
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/Mastermindzh/tidal-hifi
+        tag: 2.8.0
+        commit: 89589b75e1cdbecbeffbbe7933696bc330106d35
+        dest: main
+      - generated-sources.json
+      - type: file
+        path: run.sh
+      - type: file
+        path: com.mastermindzh.tidal-hifi.desktop
+      - type: file
+        path: com.mastermindzh.tidal-hifi.metainfo.xml
+      - type: file
+        path: icon.png
+        dest-filename: com.mastermindzh.tidal-hifi.png
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/tidal-hifi/flatpak-node/cache
+        npm_config_cache: /run/build/tidal-hifi/flatpak-node/npm-cache
+        npm_config_nodedir: /usr/lib/sdk/node16
+    subdir: main
+    build-commands:
+      - npm install
```

Comment:
change to npm install --offline for an offline build

Reply:
I have already opened an PR in the fork to fix all of the listed issues and actually take it from the binary instead of forcing users to build the application.

Reply:
@Mar0xy, 

I've looked at your branch, if you make a PR I will accept.

Thanks!